lightning/src/ln/channel.rs

```diff
@@ -58,6 +58,7 @@ use crate::sync::Mutex;
 use crate::sign::type_resolver::ChannelSignerType;
 
 use super::channel_keys::{DelayedPaymentBasepoint, HtlcBasepoint, RevocationBasepoint};
+use super::msgs::ClosingSignedFeeRange;
```

Comment:
nit: there's 3 `msgs::` imports, could we consolidate them or have one `use crate::ln::msgs;` and one import for the other imported fields?